src/components/editor/Bindings/index.tsx

```diff
@@ -49,6 +49,11 @@ function BindingsMultiEditor({
 }: Props) {
     const theme = useTheme();
 
+    const localStore = useMemo(
+        () => createEditorStore(EditorStoreNames.GENERAL),
+        []
+    );
```

Comment:
This is an easy solution to just spamming insane number of stores.

They still get spammed but not _as_ bad as before.